test/integration/models/test_lke.py

```diff
@@ -0,0 +1,122 @@
+import re
+from test.integration.helpers import get_test_label, wait_for_condition
+
+import pytest
+
+from linode_api4.errors import ApiError
+from linode_api4.objects import LKECluster, LKENodePool, LKENodePoolNode
+
+
+@pytest.fixture(scope="session")
+def create_lke_cluster(get_client):
+    node_type = get_client.linode.types()[1]  # g6-standard-1
+    version = get_client.lke.versions()[0]
+    region = get_client.regions().first()
+    node_pools = get_client.lke.node_pool(node_type, 3)
+    label = get_test_label() + "_cluster"
+
+    cluster = get_client.lke.cluster_create(region, label, node_pools, version)
+
+    yield cluster
+
+    cluster.delete()
+
+
+def test_get_lke_clusters(get_client, create_lke_cluster):
+    cluster = get_client.load(LKECluster, create_lke_cluster.id)
+
+    assert cluster._raw_json == create_lke_cluster._raw_json
+
+
+def test_get_lke_pool(get_client, create_lke_cluster):
+    pytest.skip("client.load(LKENodePool, 123, 123) does not work")
+
+    cluster = create_lke_cluster
+
+    pool = get_client.load(LKENodePool, cluster.pools[0].id, cluster.id)
+
+    assert cluster.pools[0]._raw_json == pool
+
+
+def test_cluster_dashboard_url_view(create_lke_cluster):
+    cluster = create_lke_cluster
+
+    url = cluster.cluster_dashboard_url_view()
```

Comment:
This field may not be available until the cluster is fully provisioned. Maybe we could add polling to check whether this endpoint returns a valid value within a timeout.